sdk/cosmos/azure-cosmos/docs/ErrorCodesAndRetries.md

```diff
@@ -5,7 +5,7 @@
y the <br/>read request against a second replica for the partition (in the same region) with the specified session token. </li><li>If the second replica also throws a 404: </br><ul><li>If there are additional regions for the Cosmos DB account: </br><ul><li>For a single-write region account, the client retries the request against the write region for the account (if the first request targeted the read region). The client will follow the same path of targeting 1 replica followed by a retry against another replica if the read against the first replica resulted in a 404. </li><li>If all regions have 2 replicas that returned a 404, the exception is bubbled back up to the calling code. </li></ul></li></ul></li></ul></li><li>When using all other Consistency Levels: </br><ul><li>N/A as a query operation will return an empty set instead of a Resource Not Found (404) exception. </li></ul></li></ul></li><li>For Point Read Operations: </br><ul><li>When using Eventual Consistency: </br><ul><li>The SDK sends the read request to a single replica for the partition. If the replica does not contain the data, a 404 (Resource Not Found) exception is thrown.</li><li>The Cosmos DB SDK does not retry this exception and bubbles the exception to the calling code. This is because Eventual consistency favors latency over data consistency.  </li></ul></li><li>When using Session Consistency: </br><ul><li>The SDK sends the read request to a single replica for the partition along with a session token. If the replica does not contain data that is more recent than the specified session token, a 404 (Resource Not Found) exception is thrown. </br><ul><li>The Cosmos DB SDK does retry the read request against a second replica for the partition (in the same region) with the specified session token. </li><li>If the second replica also throws a 404: </br><ul><li>If there are additional regions for the Cosmos DB account: </br><ul><li>For a single-write region account, the client retries the request against the write region for the account (if the first request targeted the read region). The client will follow the same path of targeting 1 replica followed by a retry against another replica if the read against the first replica resulted in a 404. </li><li>If all regions have 2 replicas that returned a 404, the exception is bubbled back up to the calling code. </li></ul></li><ul></li></ul></li></ul></li><li>When using Bounded Staleness and Strong Consistency: </br><ul><li>The SDK sends the read request to 2 replicas for the partition in the specified region. </li><li>If one of the 2 replicas returns a 404, data from the second replica is returned by the SDK. </li><li>If both replicas return a 404, the exception is bubbled up to the application. </li><li>The Cosmos DB SDK does NOT retry the read against a remote region is both replicas in the specified region return a 404 for the point read operation. </li></ul></li></ul></li></ul>                                                           |
```

Comment:
> For a single-write region, N/A as this exception is only encountered for point read operations

How about "... is only applicable for read operations"

Reply:
will update in next iteration

Reply:
updated